lib/yard/code_objects/namespace_mapper.rb

```diff
@@ -68,6 +73,14 @@ def separators_match
         NamespaceMapper.map_match
       end
 
+      def starts_with_separator_regex
```

Comment:
These two methods have a possible caching bug-- if `NamespaceMapper.invalidate` is called, these will not be invalidated along with `map_match`.